### PR TITLE
Hamnosys statistics

### PR DESCRIPTION
**Pull request by adrianlachowicz**

### CODE REVIEW

File: utils/make_statistics.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+        # Set figure size to plot a more data
+        if len(column_stats.index) > 20:
+            plt.figure(figsize=(15, 5))
+
+        # Create and save plot
+        plt.bar(column_stats.index, column_stats.values)
+        plt.title(column_name)
+        plt.xticks(column_stats.index)
+        plt.savefig(file_path)
```
Comment (majsylw) [inline utils/make_statistics.py:33]: you can consider to add labels for x and y axis ;-)

**Review by majsylw — APPROVED** [review]
Well done ;-)
I did some code refactor to fit our code formatting standards.

**Review by natcz — APPROVED** [review]
For me everything clear and fine

File: utils/make_statistics.py (hunk 1 above)
Comment (adrianlachowicz) [reply utils/make_statistics.py:35]: Good idea, I'll add it in the next commit.

**Pull request merged**